test_global_controller/test_global_controller_verilog_sim.py

```diff
@@ -0,0 +1,46 @@
+import magma as m
+from common.genesis_wrapper import run_genesis
+from common.util import verilog_sim_available
+from common.run_verilog_sim import irun, iverilog, run_verilog_sim
+import pytest
+import glob
+
+
+def run_verilog_regression(params):
```

Comment:
let's follow the new pattern for declaring genesis wrapper. See https://github.com/StanfordAHA/garnet/blob/master/cb/cb_genesis2.py.

Let's also include all the parameters available in `top.svp` even if we use defaults for most.

Reply:
Decided against this because we can't wrap system verilog right now. Let's just keep as a direct call to run_genesis for now.